docs/data-sources/variables.md

```diff
@@ -37,24 +36,11 @@ data "octopusdeploy_variables" "example" {
 - `is_editable` (Boolean) Indicates whether or not this variable is considered editable.
 - `is_sensitive` (Boolean) Indicates whether or not this resource is considered sensitive and should be kept secret.
 - `prompt` (Attributes List) (see [below for nested schema](#nestedatt--prompt))
+- `scope` (Attributes List) As variable names can appear more than once under different scopes, a VariableScope must also be provided (see [below for nested schema](#nestedatt--scope))
```

Comment:
Looking at the old schema docs it looks like the scope should be Required, sorry I think I mislead you on the last review.

https://github.com/OctopusDeployLabs/terraform-provider-octopusdeploy/blob/isaac/v0.22.1-dsfix-copy/docs/data-sources/variables.md

Reply:
the scope is a nested property of the variable, within a datasource the results are all readonly, I think this feels right